flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableResultImpl.java

```diff
@@ -217,15 +238,30 @@ static PrintStyle rawContent() {
 	 * print the result schema and content as tableau form.
 	 */
 	private static final class TableauStyle implements PrintStyle {
+
 		private final int maxColumnWidth;
+		private final String nullColumnStyle;
```

Comment:
rename to `nullColumnPlaceholder` ?

Reply:
nullColumnDisplayValue?

Reply:
will use `nullColumn` to keep consistent with PrintUtils